architectures

```diff
@@ -2,3 +2,4 @@ bashbrew-arch   variants
 amd64     default,alpine,onbuild,slim,stretch,wheezy
 ppc64le default,onbuild,slim,stretch
 s390x default,onbuild,slim,stretch
+arm64 default,onbuild,slim,stretch
```

Comment:
@longquan7, aarch64 is the old place; you should be looking at https://hub.docker.com/u/arm64v8/.  To know where arch images are being pushed, see [here](https://github.com/docker-library/official-images/tree/a6a23d106a7a7da48be62907e21678d582be330c#architectures-other-than-amd64).

@SimenB, we use our bashbrew tool to list the architectures of images that a given image is from like in [ruby](https://github.com/docker-library/ruby/blob/050aa8adf9109eb539307bf73b72272b15cdcaee/generate-stackbrew-library.sh#L52-L66). So the way to know what arches are supported is to check the library files since they are the source of truth.